src/napari_spatialdata/__init__.py

```diff
@@ -12,6 +12,7 @@
 
 del version, parse
 
+from napari_spatialdata._interactive import Interactive as Interactive  # noqa: E402
```

Comment:
thanks for this! why the renaming Interactive as Interactive and not just from ... import Interactive?

Reply:
mypy threw an error: `error: Module "napari_spatialdata" does not explicitly export attribute "Interactive"  [attr-defined]`. Can be fixed this way or with `__all__ = ["Interactive"]`